Header header
sensor_msgs/Image image_left
sensor_msgs/Image image_right
float64 range_distance
float64[] rtk_matrix_rotm
float64[] rtk_matrix_euler
float64 rtk_latitude
float64 rtk_longitude